src/Messaging/WhatsApp/WhatsAppMessageBird.php

```diff
@@ -78,18 +78,30 @@ public function sendWhatsAppText($to, $message, $reportUrl = null)
     public function sendWhatsAppImage($to, $imageUrl, $caption = null, $reportUrl = null)
     {
         $content = new Content();
-        $content->image = [
+
+        $data = [
             'url'       => $imageUrl,
             'caption'   => $caption,
         ];
 
+        $type = 'image';
+
+        if(@exif_imagetype($imageUrl)) {
```

Comment:
```suggestion
        if(@exif_imagetype($imageUrl)) {
```
why @ is included bro?

Reply:
to prepend error which is caused by using that method bro